### PR TITLE
Youtube video page

### PR DESCRIPTION
**Pull request by mohapakram**
## In this feature we added:
* An embedded youtube video player.
* Header with logo and  text input for search
* Aside column with recommended videos

### CODE REVIEW

**Comment by peterrobert** [comment]
**PULL REQUEST UNDER REVIEW BY PETER ROBERT**

File: Readme.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ Open index.html file in your browser.
 
 - Github: [@githubhandle](https://github.com/mohabakram)
 
-👤 **Author2**
+👤 **Olawale**
 
-- Github: [@githubhandle](https://github.com/githubhandle)
+- Github: [@githubhandle](https://github.com/OlawaleJoseph)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - Linkedin: [linkedin](https://linkedin.com/linkedinhandle)
```
Comment (peterrobert) [inline Readme.md:29]: Add more description to your readme file. You can use this [template](https://github.com/microverseinc/readme-template) to make your readme more descriptive

File: index.html [hunk 2]
```diff
@@ -40,23 +40,147 @@
...
+            </div>
+            <div class="reactions">
+              <div class="likes">
+                <i class="fas fa-thumbs-up"><span> 554</span></i>
+                <i class="fas fa-thumbs-down"> <span>213</span> </i>
+              </div>
+              <i class="fas fa-share"><span>share</span></i>
+              <i class="fas fa-list-ul"><span>save</span></i>
+              <i class="fas fa-bars"></i>
+            </div>
```
Comment (peterrobert) [inline index.html]: Kindly use buttons. It's okay to use font-awesome icons but you should make them clickable. You can do this by changing the cursor in your css

**Review by peterrobert — CHANGES_REQUESTED** [review]
**CHANGES REQUIRED** :stop_sign: 

Great work so far :clap:  But in order for me to approve this project you need to make the following changes

1. You need to fix all the conflicts between your branches.

2. You need to create your pull request messages like described [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

3. Remember to use <section> tag for your different sections in your webpage

4. Fix all HTML validation errors. You can validate your HTML [here](https://validator.w3.org/) 
html

5. Fix the alignment of the UpNext column. The title and the text aren't properly aligned.
allign

happy coding :computer:  and remember to submit another review once you fix all the recommended issues:smiley:

**Comment by mohapakram** [comment]
@peterrobert hello peter,
We have only one conflict with the master branch and I don't think we can resolve it without merging the feature branch and the master branch, so when we get the pull request approved we will fix the conflict and merge.

If there is a way to fix the conflict without merging please let us know.

thank you for the awesome review!

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @mohapakram and @OlawaleJoseph, great job :+1: We are almost there. Let us fix a few comments so we can approve the project. Kindly check comments below:
- The avatar is not positioned correctly, let us fix it. Screenshot below
noww
- The subscribe button should be a link
ssb

Kindly make these changes and submit another review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved  ✅ 
****
### Hi @mohapakram and @OlawaleJoseph 👋 👋 
Great job implementing the requested changes! 👏 👏 👏  It's a 👍 from me!  
Go ahead and merge this pull request and keep up the good work 😄 

- Don't forget to resolve the conflicts before merging 😉 

Happy coding! 😄

**Pull request merged**